android/src/main/java/com/reactnativestripesdk/PaymentMethodCreateParamsFactory.kt

```diff
@@ -422,11 +424,31 @@ class PaymentMethodCreateParamsFactory(
       )
   }
 
+  @Throws(PaymentMethodCreateParamsException::class)
+  private fun createUSBankAccountPaymentSetupParams(): ConfirmSetupIntentParams {
+    params.getString("accountNumber")?.let {
+      if (billingDetailsParams?.name.isNullOrBlank()) {
+        throw PaymentMethodCreateParamsException("When creating a US bank account payment method, you must provide the following billing details: name")
+      }
+      return ConfirmSetupIntentParams.create(
+        paymentMethodCreateParams = createUSBankAccountParams(params),
+        clientSecret = clientSecret,
+      )
+    } ?: run {
+      // Payment method is assumed to be already attached through via collectBankAccountForSetup
```

Comment:
@jameswoo-stripe I think it's fine, but can we double check we can make this assumption?

Reply:
Went through the flow with @charliecruzan-stripe, it looks good to me